pkg/operator/operator/k8s.go

```diff
@@ -388,6 +388,10 @@ func getEnvVars(api *spec.API, container string) []kcore.EnvVar {
 				Name:  "CORTEX_PROJECT_DIR",
 				Value: path.Join(_emptyDirMountPath, "project"),
 			},
+			kcore.EnvVar{
+				Name:  "S6_BEHAVIOUR_IF_STAGE2_FAILS",
```

Comment:
Tying into what David has said, I think we can now remove all `if [ $status -ne 0 ]` checkers in the bootloader. Those ifs were only in because the captured signals did not kill the pod. We should test this.